projects/storefrontlib/cms-components/myaccount/address-book/address-form/address-form.component.html

```diff
@@ -192,7 +192,9 @@
           </ng-container>
         </ng-container>
       </div>
-      <div class="form-group">
+
+      <!-- TODO: (#CXSPA-53) Remove feature config check in 6.0. -->
+      <div class="form-group col-md-6" *cxFeatureLevel="'!5.2'">
```

Comment:
Should we have this `col-md-6` class added here?

Reply:
removed `col-md-6` since it wasn't there originally